cluster/up.sh

```diff
@@ -20,3 +20,11 @@ set -e
 source hack/common.sh
 source cluster/$KUBEVIRT_PROVIDER/provider.sh
 up
+
+# Wait until default ovs-cni plugin appears
```

Comment:
I may miss something but what are these commands for? You create ds, then you delete it. Is it some kind of test script? The name - up.sh - suggests it should deploy the plugin, not tear it down.